tmt/utils.py

```diff
@@ -1695,6 +1695,13 @@ def __init__(
         self.check_success = check_success
 
 
+class RetryError(GeneralError):
+    """ Retries unsuccessful """
+
+    def __init__(self, label: str, exc: Optional[Exception]) -> None:
+        super().__init__(f"Retries of {label} unsuccessful. {exc}")
```

Comment:
If there is an exception that caused this one, it shouldn't be passed to `RetryError` as an argument, the exception chaining is the way to connect exceptions:

```python
raise RetryError(...) from exc
```

Therefore the `exc` parameter is not needed.